satellite/marketing/marketingweb/server.go

```diff
@@ -8,18 +8,31 @@ import (
 	"html/template"
 	"net"
 	"net/http"
+	"time"
+	"reflect"
 	"path/filepath"
 
 	"github.com/zeebo/errs"
 	"go.uber.org/zap"
 	"golang.org/x/sync/errgroup"
+	"github.com/gorilla/schema"
+	"storj.io/storj/satellite/marketing"
 	"github.com/gorilla/mux"
 )
 
-// Error is satellite marketing error type
-var Error = errs.Class("satellite marketing error")
+// Time converter is used by decoder to format expiration date field from form input
+var (
+	Error       = errs.Class("satellite marketing error")
+	decoder     = schema.NewDecoder()
```

Comment:
Is it thread-safe?

Reply:
Same thoughts as https://github.com/storj/storj/pull/2186#discussion_r293191793.

Reply:
I localized decoder into the only block it was needed. I believe that solves the concern. Correct me if i'm wrong.